tests/api/test_view.py

```diff
@@ -129,28 +127,6 @@ def mocked_execute(*args, **kwargs):
     assert content["errors"][0]["message"] == "Spanish inquisition"
 
 
-class LoggingHandler(logging.Handler):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.messages = []
-
-    def emit(self, record: logging.LogRecord):
-        exc_type, exc_value, _tb = record.exc_info
-        self.messages.append(
-            f"{record.name}[{record.levelname.upper()}].{exc_type.__name__}"
-        )
-
-
-@pytest.fixture
-def graphql_log_handler():
```

Comment:
Why removing this code?

Reply:
Not removing, moving to `conftest.py`

Reply:
Ahh sorry, missed that 👀

Reply:
NP, btw I can revert that but it looks like sth that could be used in other tests so I decided to move it up. I can revert it if You want.